api/schemas/report.json

```diff
@@ -146,6 +147,11 @@
             "type":        "boolean"
         },
 
+        "noop_pending": {
+            "description": "Whether noop resource events exist",
```

Comment:
I'd hope that elsewhere in the report schema was specify what a "noop event" is, but maybe it's worthwhile to wordsmith this a little bit to indicate that it means "there are changes that we decided not to apply because of noop". @jtappa what are your thoughts here?